### ISSUE TITLE
Primeng in Angular 11

### ISSUE THREAD

**Issue by mobius127** [primefaces/primeng#9544]
Hi, I'm trying to instal primeng in Angular 11

PS C:\_Projects\refineries> npm install primeng --save
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: refineries@0.0.0
npm ERR! Found: @angular/common@11.0.1
npm ERR! node_modules/@angular/common
npm ERR!   @angular/common@"~11.0.1" from the root project
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer @angular/common@"^7.0.0 || ^8.0.0 || ^9.0.0 || ^10.0.0" from primeng@11.0.0-rc.1
npm ERR! node_modules/primeng
npm ERR!   primeng@"*" from the root project
npm ERR!
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.

**Comment by yuanqingshan** [comment]
这么取消订阅邮件

Javier González <notifications@github.com> 于2020年11月19日周四 下午6:44写道：

**Comment by mobius127** [comment]
This is the entire log

# npm resolution error report

2020-11-19T10:49:16.813Z

While resolving: refineries@0.0.0
Found: @angular/common@11.0.1
node_modules/@angular/common
  @angular/common@"~11.0.1" from the root project

Could not resolve dependency:
peer @angular/common@"^7.0.0 || ^8.0.0 || ^9.0.0 || ^10.0.0" from primeng@11.0.0-rc.1
node_modules/primeng
  primeng@"^11.0.0-rc.1" from the root project

Fix the upstream dependency conflict, or retry
this command with --force, or --legacy-peer-deps
to accept an incorrect (and potentially broken) dependency resolution.

Raw JSON explanation object:

{
  "code": "ERESOLVE",
  "current": {
    "name": "@angular/common",
    "version": "11.0.1",
    "whileInstalling": {
      "name": "refineries",
      "version": "0.0.0",
      "path": "C:\\_Projects\\refineries"
    },
    "location": "node_modules/@angular/common",
    "dependents": [
      {
        "type": "prod",
        "name": "@angular/common",
        "spec": "~11.0.1",
        "from": {
          "location": "C:\\_Projects\\refineries"
        }
      }
    ]
  },
  "edge": {
    "type": "peer",
    "name": "@angular/common",
    "spec": "^7.0.0 || ^8.0.0 || ^9.0.0 || ^10.0.0",
    "error": "INVALID",
    "from": {
      "name": "primeng",
      "version": "11.0.0-rc.1",
      "whileInstalling": {
        "name": "refineries",
        "version": "0.0.0",
        "path": "C:\\_Projects\\refineries"
      },
      "location": "node_modules/primeng",
      "dependents": [
        {
          "type": "prod",
          "name": "primeng",
          "spec": "^11.0.0-rc.1",
          "from": {
            "location": "C:\\_Projects\\refineries"
          }
        }
      ]
    }
  },
  "peerConflict": null,
  "strictPeerDeps": false,
  "force": false
}

**Comment by cagataycivici** [comment]
Please use forum for questions and help requests.

**Comment by micobarac** [comment]
Why was this issue closed? I am facing the same warning when installing the Primeng with Angular 11.

```
warning " > primeng@11.0.0-rc.1" has incorrect peer dependency "@angular/core@^7.0.0 || ^8.0.0 || ^9.0.0 || ^10.0.0".
warning " > primeng@11.0.0-rc.1" has incorrect peer dependency "@angular/common@^7.0.0 || ^8.0.0 || ^9.0.0 || ^10.0.0".
warning " > primeng@11.0.0-rc.1" has incorrect peer dependency "@angular/forms@^7.0.0 || ^8.0.0 || ^9.0.0 || ^10.0.0".
```

**Comment by pete-mcwilliams** [comment]
There is a problem in the released primeng@11.0.0-rc.1

npm install primeng
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: angular-tailwind-example@0.0.0
npm ERR! Found: @angular/common@11.0.2
npm ERR! node_modules/@angular/common
npm ERR!   @angular/common@"~11.0.2" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer @angular/common@"^7.0.0 || ^8.0.0 || ^9.0.0 || ^10.0.0" from primeng@11.0.0-rc.1
npm ERR! node_modules/primeng
npm ERR!   primeng@"*" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR! 
npm ERR! See /Users/pmcw/.npm/eresolve-report.txt for a full report.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/pmcw/.npm/_logs/2020-11-28T10_11_50_645Z-debug.log
